.travis.yml

```diff
@@ -1,11 +1,15 @@
 language: go
+sudo: false
 
 go_import_path: go.uber.org/fx
-
 go:
   - 1.8.x
   - tip
```

Comment:
I can definitely add `1.9.x`. I'd prefer not to allow tip to fail unless it becomes flaky.